bob/scheduler.py

```diff
@@ -25,9 +25,9 @@ def __init__(self, updater: Updater):
                                                 start=True,
                                                 tz=tz)
 
-        cron_friday_noon = '0 8 * * *'  # “Every day at 08:00.”
-        self.friday_noon_task = aiocron.crontab(str(cron_friday_noon),
-                                                func=self.friday_noon,
+        cron_every_morning = '0 8 * * *'  # “Every day at 08:00.”
+        self.friday_noon_task = aiocron.crontab(str(cron_every_morning),
+                                                func=self.good_morning_broadcast,
```

Comment:
Do we want to send this message every morning? At least opt-in and not opt-out maybe?